src/components/admonition/admonition.types.ts

```diff
@@ -27,3 +27,41 @@ export type AdmonitionType =
     | 'example'
     | 'quote'
     | 'question';
+
+const admonitionIcons: Record<string, string> = {
+    danger: 'poison',
+    info: 'info',
+    note: 'appointment_reminders',
+    tip: 'idea',
+    warning: 'error',
+    success: 'ok',
+    failure: 'high_priority',
+    example: 'test_tube',
+    quote: 'quote_left',
+    question: 'help',
+};
+
+const admonitionHeadings: Record<string, string> = {
+    danger: 'Danger',
+    info: 'Info',
+    note: 'Note',
+    tip: 'Tip',
+    warning: 'Warning',
+    success: 'Success',
+    failure: 'Failure',
+    example: 'Example',
+    quote: 'Quote',
+    question: 'Question',
+};
+
+export function getIcon(type: string): string {
+    return admonitionIcons[type] || 'info';
+}
+
+export function getHeading(heading: string): string {
+    if (heading) {
+        return heading;
+    }
+
+    return admonitionHeadings[heading] || 'Info';
+}
```

Comment:
This file should only contain types. Helper functions needs to be moved to a separate file

Reply:
OK. I'll fix ;)